src/mlpack/core/tree/cover_tree/traits.hpp

```diff
@@ -33,6 +33,12 @@ class TreeTraits<CoverTree<MetricType, StatisticType, MatType, RootPointPolicy>>
   static const bool HasOverlappingChildren = true;
 
   /**
+   * Cover trees do have self-children, so points can be included in more than
+   * one node.
+   */
+  static const bool HasDuplicatedPoints = true;
```

Comment:
Hi @rcurtin ,
In fact we use another flag: `UniqueNumDescendants`,  to recognize the case where `NumDescendants()` includes repeated points.
The flag `HasDuplicatedPoints` is not used anywhere, in fact I orginally removed it but you suggested in https://github.com/mlpack/mlpack/pull/743#discussion_r73456217 that you would like to keep it for the future.
Would you prefer to remove it or keeping it for future use?